src/common/datomish/query/clauses.cljc

```diff
@@ -204,17 +209,16 @@
   ;; Otherwise -- perhaps each leg of the `or` binds different variables, which
```

Comment:
Might as well add comment about complex `or` here.  It's not clear to me that it's worth separating simple from complex; the former must be a subset of the latter.  But that's for you to decide.